wgsl/index.bs

```diff
@@ -8228,19 +8243,30 @@ Attributes are used for a variety of purposes such as specifying the interface w
 Generally speaking, from the language's point-of-view, attributes can be
 ignored for the purposes of type and semantic checking.
 Additionally, the attribute name is a [=context-dependent name=], and
-some attribute parameters are also context-dependent names.
+some attribute parameters are also context-dependent names. Attributes in WGSL have the
```

Comment:
Tiniest of nits: prefer sentence on a new line. I find it makes future edits cleaner in diffs.

Reply:
Makes sense, made a commit, PTAL! Thank you!